### PR TITLE
Enable simulator to run HE

### PR DESCRIPTION
**Pull request by yhwen**
Fixes # .

### Description

Added the SecurityContentService.initialize() to enable simulator to run HE.

Usage: 

- Use the provision tool to generate the server and client tenseal files.
- Place the tenseal files into the "startup" sub-folder in the simulator workspace.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/private/fed/app/simulator/simulator_worker.py [hunk 1]
```diff
@@ -245,6 +247,10 @@ def main(args):
     # AuditService.initialize(audit_file_name=WorkspaceConstants.AUDIT_LOG)
     AuditService.the_auditor = SimulatorAuditor()
 
+    os.makedirs(os.path.join(args.workspace, WorkspaceConstants.STARTUP_FOLDER_NAME), exist_ok=True)
```
[the remark this answers is not in the document]
Comment (yhwen) [reply nvflare/private/fed/app/simulator/simulator_worker.py]: Extract the common codes to init_security_content_service()

**Comment by holgerroth** [comment]
Should we add some instructions on how to do this?

**Pull request merged**